packages/table/src/table.tsx

```diff
@@ -372,7 +372,11 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const { isRowHeaderShown } = this.props;
         this.validateGrid();
         return (
-            <div className={Classes.TABLE_CONTAINER} ref={this.setRootTableRef} onScroll={this.handleRootScroll}>
+            <div
+                className={classNames(Classes.TABLE_CONTAINER, this.props.className)}
```

Comment:
not to be a stickler but both of these render functions deconstruct `this.props` a few lines up. you could add `className` there?